### PR TITLE
Make colorbar ticks format in generics plot configurable

### LINKED ISSUE #1828: colorbar labels need better configurability
colorbar label ticks cannot be properly configured.
Also, VSEC and HSEC plots behave inconsistent.

### PR DESCRIPTION
**Pull request by joernu76**
Fix #1828

### CODE REVIEW

File: mslib/mswms/generics.py [hunk 1]
```diff
@@ -303,6 +303,31 @@ def get_log_levels(cmin, cmax, levels=None):
...
 
+
+CBAR_LABEL_FORMATS = {
+    "log": "%.3g",
+    "log_ice_cloud": "%.0E",
+}
+
+
+def get_cbar_label_format(style, maxvalue):
+    if style in CBAR_LABEL_FORMATS:
```
Comment (ReimarBauer) [inline mslib/mswms/generics.py:313]: we have humanfriendly in our package. Does this what you want?

https://humanfriendly.readthedocs.io/en/latest/readme.html#getting-started

Comment (joernu76) [reply mslib/mswms/generics.py:313]: Maybe. I'll have a look at that. The major part of this PR is, however, to put it under user control, not the actual behaviour, which human-frienldy likely improves.

Comment (joernu76) [reply mslib/mswms/generics.py:313]: Nope. Sadly, humanfriendly does something differently very well.  Close but not close enough.

**Review by ReimarBauer — APPROVED** [review]
thx

**Pull request merged**